var/spack/repos/builtin/packages/py-pyqt4/package.py

```diff
@@ -14,18 +14,30 @@ class PyPyqt4(SIPPackage):
 
     homepage = "https://www.riverbankcomputing.com/software/pyqt/intro"
     url      = "http://sourceforge.net/projects/pyqt/files/PyQt4/PyQt-4.12.3/PyQt4_gpl_x11-4.12.3.tar.gz"
-    import_modules = ['PyQt4']
+
+    sip_module = 'PyQt4.sip'
+    import_modules = [
+        'PyQt4', 'PyQt4.Qt', 'PyQt4.QtCore', 'PyQt4.QtDeclarative',
+        'PyQt4.QtDesigner', 'PyQt4.QtGui', 'PyQt4.QtHelp',
+        'PyQt4.QtMultimedia', 'PyQt4.QtNetwork', 'PyQt4.QtOpenGL',
+        'PyQt4.QtScript', 'PyQt4.QtScriptTools', 'PyQt4.QtSql', 'PyQt4.QtSvg',
+        'PyQt4.QtTest', 'PyQt4.QtWebKit', 'PyQt4.QtXml', 'PyQt4.QtXmlPatterns'
+    ]
 
     version('4.12.3', sha256='a00f5abef240a7b5852b7924fa5fdf5174569525dc076cd368a566619e56d472')
     version('4.11.3', '997c3e443165a89a559e0d96b061bf70',
             url='http://sourceforge.net/projects/pyqt/files/PyQt4/PyQt-4.11.3/PyQt-x11-gpl-4.11.3.tar.gz')
 
     # Supposedly can also be built with Qt 5 compatibility layer
-    depends_on('qt@:4+phonon+dbus')
+    depends_on('qt@:4')
```

Comment:
@Sinan81 you said you got this package working with `qt@:4~phonon~dbus`? Just want to confirm before I remove this requirement.

Reply:
Will do before end of the day. I am pretty sure 'phonon' is optional.

Reply:
I confirm that py-pyqt4 builds with `qt@4.8.7 ~phonon~dbus`
the specific qt installation which it depends on has the following build options
```
$ spack find -lv qt /st2
==> 1 installed package
-- linux-centos7-x86_64 / gcc@8.2.0 -----------------------------
st2p2z5 qt@4.8.7~dbus~examples~framework freetype=spack ~gtk~krellpatch~opengl patches=4f40b7c6230471c19fd1aea45e264a1642c654023d2da1c8063a05ef095d6586,88ec914c9f2026599cce3258006db3fab48138cb9b37f0a7a182d6731fbe2d4d,a4f37427b34d88df6a6adc16e1e6c805087d4f3a7c658e1797450b0912fcbf5d ~phonon+shared+sql+ssl+tools~webkit
```